library/build.gradle

```diff
@@ -15,12 +15,12 @@ repositories {
 }
 
 android {
-    compileSdkVersion 22
-    buildToolsVersion "22.0.1"
+    compileSdkVersion 25
+    buildToolsVersion '25.0.2'
 
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 22
+        targetSdkVersion 25
```

Comment:
That would only happen in the (very unlikely) case in which the app didn't specify it itself. But yeah we could get rid of it.